Shared/Theme/ThemeManager.swift

```diff
@@ -41,7 +41,7 @@ final public class DefaultThemeManager: ThemeManager, Notifiable {
     private var userDefaults: UserDefaultsInterface
     private var mainQueue: DispatchQueueInterface
 
-    public var window: UIWindow?
+    public var windows: [UIWindow]?
```

Comment:
What is the thinking behind updating this to use an array? I believe we currently only ever have one window. Even in the future when we add multi-window support we will still only have one window per scene so we will still need a different solution for that use case.